app/controllers/herbaria/shared_private_methods.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+# private methods shared by HerbariaController and subcontrollers
+module Herbaria::SharedPrivateMethods
```

Comment:
Is there a better or more typical way to share private actions among `HerbariaController` and namespaced controllers?

Reply:
Hm. Seems like the first ones (`keep_track_of_referrer` and `redirect_to_referrer`) could maybe be inherited from Application_Controller.